dashboard/app/helpers/aichat_safety_helper.rb

```diff
@@ -80,6 +92,45 @@ def find_toxicity(role, text, locale)
     private def get_safety_system_prompt
       'You are a content filter trying to keep a school teacher out of trouble. Determine if chat text is inappropriate for an American public middle school classroom. Examples of inappropriate content: profanity, swears, illegal behavior, insults, bullying, slurs, sex, violence, racism, sexism, threats, weapons, dirty slang, etc. If text is inappropriate, respond with the single word `INAPPROPRIATE`, otherwise respond with the single word `OK`.'
     end
+
+    private def get_safety_system_prompt_version
+      'V0'
+    end
+
+    private def report_openai_safety_check(metric_name)
+      Cdo::Metrics.push(SharedConstants::AICHAT_METRICS_NAMESPACE,
+        [
+          {
+            metric_name: metric_name,
+            value: 1,
+            unit: 'Count',
+            timestamp: Time.now,
+            dimensions: [
+              {name: 'Environment', value: CDO.rack_env},
+              {name: 'SafetySystemPrompt', value: get_safety_system_prompt},
```

Comment:
nit: `SafetySystemPromptVersion` (or `PromptVersion`/similar for a shorter name)?